app/src/main/java/com/ably/tracking/demo/subscriber/ui/screen/createorder/CreateOrderScreenState.kt

```diff
@@ -0,0 +1,17 @@
+package com.ably.tracking.demo.subscriber.ui.screen.createorder
+
+import com.ably.tracking.demo.subscriber.common.canParseToDouble
+
+data class CreateOrderScreenState(
+    val trackableId: String = "",
+    val fromLatitude: String = "51.1065859",
+    val fromLongitude: String = "17.0312766",
+    val toLatitude: String = "51.1065859",
+    val toLongitude: String = "17.0312766"
```

Comment:
Is it okay to use those hardcoded values? Wouldn't it be better to show something like `NA`? :thinking:

Reply:
I've put those to make it easier to just run it, instead of forcing users to enter anything

Reply:
Ohh okay, I thought that this is rendered on the UI :sweat_smile: It all makes sense now, thanks :bow: